dotcom-rendering/src/web/browser/islands/doHydration.ts

```diff
@@ -17,7 +16,7 @@ import { initPerf } from '../initPerf';
  */
 export const doHydration = (
 	name: string,
-	data: Attributes | null,
```

Comment:
I'm not sure how we get away with `Attributes` here - but it's not actually accurate to the types